src/highdicom/sr/content.py

```diff
@@ -730,6 +733,28 @@ def __init__(
                 )
                 self.ContentSequence.append(modifier_item)
 
+    @property
+    def topographical_modifier(self) -> Union[CodedConcept, None]:
+        matches = find_content_items(
+            self,
+            name=codes.SCT.TopographicalModifier,
+            value_type=ValueTypeValues.CODE
+        )
+        if len(matches) > 0:
+            return matches[0].value
```

Comment:
Throw an error or log a warning? There should only be one item according to the standard. If there is more than one, it may still be valuable to return the first item without stopping program execution. What do you think?

Reply:
No strong feelings, a warning is probably sufficient

Reply:
Added log statements via 4475f05450ed030f1619dc741bc701e2f68b555b